app/migrations/seed.py

```diff
@@ -355,7 +355,10 @@ def create_master_organisation(reserve_token):
         print('Creating master organisation')
         if reserve_token:
             print('Binding to reserve token')
-        master_organisation = Organisation(name='Reserve', is_master=True, token=reserve_token)
+        master_organisation = Organisation(
+            name='Reserve', is_master=True,
+            token=reserve_token, country_code=app.config.get('DEFAULT_COUNTRY', 'AU')
+        )
```

Comment:
question here @enjeyw - app doesn't seem to be defined when running `quick_setup_script.sh` locally (which is fine as fallback can just be 'AU'). Just want to double-check logic for a production setup

Reply:
Oh cool, saved by thy test https://circleci.com/gh/teamsempo/SempoBlockchain/4554

Reply:
Fixed manage.py